examples/terraform/aws/variables.tf

```diff
@@ -19,7 +19,7 @@ variable "cluster_name" {
 }
 
 variable "aws_region" {
-  default     = "eu-west-3"
+  default     = "eu-central-1"
```

Comment:
Why we changed back to `eu-central-1`? I think we had less number of flakes on `eu-west-3`.

Reply:
The point was that automated clean-up of leftovers job is working in `eu-central-1`, but OK, reverted back to Paris.